CMakeLists.txt

```diff
@@ -157,7 +157,7 @@ else()
   endif()
 
   set(TRITON_ONNXRUNTIME_DOCKER_IMAGE "tritonserver_onnxruntime")
-  set(TRITON_ONNXRUNTIME_DOCKER_MEMORY "8g")
+  set(TRITON_ONNXRUNTIME_DOCKER_MEMORY "$<IF:$<BOOL:WIN32>,32g,8g>")
```

Comment:
Does this still work if a windows machine doesn't have 32g? Does the compilation just take all the available memory instead?

Reply:
It's a docker option if it can't allocate given amount it should use default limits. 
Because it's a cached argument user can always change it as per their needs.